alembic/operations/ops.py

```diff
@@ -2440,7 +2441,7 @@ class ExecuteSQLOp(MigrateOperation):
 
     def __init__(
         self,
-        sqltext: Union[Update, str, Insert, TextClause],
+        sqltext: Union[Delete, Insert, Update, TextClause, str],
```

Comment:
this should be more general, such as https://github.com/sqlalchemy/sqlalchemy/blob/f23c617e1060fc44c90528d2bfe7a6bd77f5d010/lib/sqlalchemy/sql/base.py#L1004 unioned with string

Reply:
plus there are some scripts that generate the op.pyi stub and such, this will be needed to pass the CI build

Reply:
Updated